app/mixins/create_Inventory.js

```diff
@@ -0,0 +1,19 @@
+import Ember from "ember";
```

Comment:
Why caps `I` for file name i.e `create_Inventory`?

Reply:
There is no need to create this mixins. You can directly call this service from `.hbs`.
easiest way could be `{{action 'actionName' params target=packageService}}`